packages/web-components/src/components/rux-table/rux-table-cell/rux-table-cell.scss

```diff
@@ -1,11 +1,21 @@
 :host {
     display: table-cell;
-    border-width: 1px 0;
+    vertical-align: middle;
+    border-width: 0 0 1px 0;
     border-style: solid;
     border-color: var(--color-background-base-default);
-    padding: 0.625rem 1rem;
+    font-family: var(--font-body-1-font-family);
+    font-size: var(--font-body-1-font-size);
+    font-weight: var(--font-body-1-font-weight);
+    letter-spacing: var(--font-body-1-letter-spacing);
+    line-height: var(--font-body-1-line-height);
+    padding: var(--spacing-0) var(--spacing-2);
+    height: calc(var(--spacing-8) - 1px); //32 (height and borders)
```

Comment:
I had some deep discussions with design while you were gone and I learned that when there is documentation in the design that indicates a fixed height, this is a design directive, not a dev one. I took a look at the designs in Figma and it does specify this height, but it says minimum, so perhaps change this to min-height? Hopefully that doesn't mess up the styling.

Reply:
I had it at min height at first but display:table doesn't respect min-height like other displays do. It treats height as min-height as far as I know. :)